src/renderer/src/setup/sentryReact.ts

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { createBrowserRouter } from 'react-router-dom'
+import * as ElectronRedererSentry from '@sentry/electron/renderer'
+import * as ReactSentry from '@sentry/react'
+
+import { EnvironmentMode } from '../../../common/EnvironmentMode'
+import { sentryCommonConfig } from '../../../common/GlobalConfig'
+
+const isProductionMode = (): boolean => {
+  const env = window['env']
+  return env && env['SENTRY_DNS'] && env['ENVIRONMENT'] && env['ENVIRONMENT'] == EnvironmentMode.PROD
+}
+export function setupSentryReact() {
+  if (isProductionMode()) {
+    ElectronRedererSentry.init(
+      {
+        dsn: window['env']['SENTRY_DNS'],
```

Comment:
```suggestion
        dsn: window.env.SENTRY_DNS
```

You will need to type in the src/renderer/src/@types/window.ts file